examples/README.md

```diff
@@ -0,0 +1,59 @@
+# Example documentation WIP
+
+## `docking-protein-protein/docking-protein-protein-mpi`
+
+To run this example you must have `mpi4py` installed in the haddock3 python environment, and `OpenMPI` in the host system.
+
+
+```bash
+$ pip install mpi4py
+# or
+$ conda install -c conda-forge mpi4py
+```
+
+**Do not run it as: `mpirun -np haddock3 run.cfg`**, this logic is handled internally.
+
+Edit the `.cfg` according to the `.job` parameters you have set
```

Comment:
From what I understand, the user needs to create the `job` files him/herselves. Right? But what you described seems to be straightforward to add it to the python shell. Can't you make this all go automatic under the hood when the user-defined in the config file `mode=mpi`?

I was reviewing the explanation. I found it not clear to understand how to combine the use of the `haddock3 CONFIG`, with the `sbatch docking-protein-protein-mpi.job` **and** the new client.

Can those `SBATCH` parameters be part of the new client, instead?

Reply:
yes and no, this is a highly complex scenario that involves particular cluster configurations. I see your point, its possible to add:

```toml
nodes = 3
tasks_per_node = 96
```

In the `.cfg` file and then generate the `.job` on the fly, but if you are using MPI for this high throughput processing you are an experienced HPC user and will very likely need some extra steps and then our `.job` is useless. So i'd say its not the best idea to "facilitate" this.